<launch>
	<!-- 启动仿真环境 -->
	<include file="$(find prometheus_gazebo)/launch_basic/sitl_outdoor_1uav_P450.launch">
		<arg name="world" default="$(find prometheus_gazebo)/gazebo_worlds/detection_worlds/KKSwarmCar/kkcar.world" />
		<arg name="sdf" value="$(find prometheus_gazebo)/gazebo_models/uav_models/p450_monocular/p450_monocular.sdf" />
	</include>

	<!-- 启动prometheus控制器 -->
	<include file="$(find prometheus_uav_control)/launch/uav_control_main_outdoor.launch">
		<arg name="launch_prefix" default="bash -c 'gnome-terminal --tab -- $0 $@'" />
	</include>



	<!-- 启动跟踪程序 -->
	<node pkg="prometheus_demo" type="Tracking_car" name="Tracking_car" output="screen" launch-prefix="bash -c 'sleep 2; gnome-terminal --tab -- $0 $@; bash'">
		<param name="uav_id" value="1" />
		<!-- 相机安装偏差 -->
		<param name="camera_offset_x" value="0.0" />
		<param name="camera_offset_y" value="0.0" />
		<param name="camera_offset_z" value="-0.1" />
		<!-- 追踪控制参数 -->
		<param name="kpx_track" value="0.3" />
		<param name="kpy_track" value="0.3" />
		<param name="kpz_track" value="0.05" />
		<!-- 跟踪距离保持 -->
		<param name="keep_x_distance" value="0.0" />
		<param name="keep_y_distance" value="0.0" />
		<param name="keep_z_distance" value="1.8" />
	</node>

	<node pkg="rqt_image_view" type="rqt_image_view" name="rqt_image_view"></node>
</launch>
